src/main/resources/application.yml

```diff
@@ -43,8 +43,11 @@ messaging:
             password: guest
             virtualHost: "/"
             camelRoute:
-                destinationName: engineEvents
+                destinationName: acsCloudConnectorUpdateMetadata
```

Comment:
It's very confusing with both of these in here.  Can the old `destinationName` and `toRoute` be removed?